devtools/packaging/scripts/source/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Build script for Linux distribution, for use in automated packaging.
+# Note that this must be run from outside the checked-out openmm/ directory.
+
+# Set relative workspace path.
+export WORKSPACE=`pwd`
+
+# Add conda binaries to path.
+PATH=$WORKSPACE/miniconda/bin:$PATH
+
+INSTALL=`pwd`/install
+if [ -e $INSTALL ]; then
+    rm -rf $INSTALL
+fi
+
+CMAKE_FLAGS="-DCMAKE_INSTALL_PREFIX=$INSTALL"
+
+# setting the rpath so that libOpenMMPME.so finds the right libfftw3
+#CMAKE_FLAGS+=" -DCMAKE_INSTALL_RPATH=.."
+CMAKE_FLAGS+=" -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++"
+CMAKE_FLAGS+=" -DOPENMM_BUILD_AMOEBA_CUDA_LIB=OFF"
+CMAKE_FLAGS+=" -DOPENMM_BUILD_CPU_LIB=OFF"
```

Comment:
Because we're just packaging the source distribution but still need to compile ALL OF OPENMM just to build the docs.  Only the docs and Python wrappers are taken from the build.  There is no "build only the docs" mode for OpenMM, though there is a feature request for this: https://github.com/pandegroup/openmm/issues/677